<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13pt;
color:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>一键搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;微软雅黑&quot;;
background-color: rgb(255, 180,0);
border-radius:15px;
</string>
    </property>
    <property name="text">
     <string> 最新问题</string>
    </property>
   </widget>
   <widget class="QPushButton" name="askQuesButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>提问</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userDataButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>个人资料</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>701</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>刷新界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signoutButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(255, 50, 50);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="focusButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>我的关注</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastpageButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextpageButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homepageButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endpageButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>尾页</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageNum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>562</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotopageButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>按时间搜索</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="followedButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>我的粉丝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noticeButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>通知消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13pt;
color:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>一键搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;微软雅黑&quot;;
background-color: rgb(255, 180,0);
border-radius:15px;
</string>
    </property>
    <property name="text">
     <string> 最新问题</string>
    </property>
   </widget>
   <widget class="QPushButton" name="askQuesButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>提问</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userDataButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>个人资料</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>701</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>刷新界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signoutButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(255, 50, 50);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="focusButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>我的关注</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastpageButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextpageButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homepageButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endpageButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>尾页</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageNum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>562</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotopageButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>按时间搜索</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="followedButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>我的粉丝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noticeButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
    </property>
    <property name="text">
     <string>通知消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
>>>>>>> cfdf638c3bdfa8efdbbb0c911ad54fe243c63989
